alternator/executor.cc

```diff
@@ -1496,6 +1497,23 @@ bytes extract_from_attrs_column_computation::compute_value(const schema&, const
     on_internal_error(elogger, "extract_from_attrs_column_computation::compute_value called without row");
 }
 
+// Because `CreateTable` request creates GSI/LSI together with the base table (so the base table is empty),
+// we can skip view building process and immediately mark the view as built on all nodes.
+//
+// However, we can do this only for tablet-based views because `view_building_worker` will automatically propagate
+// this information to `system.built_views` table (see `view_building_worker::update_built_views()`).
+// For vnode-based views, `view_builder` will process the view as mark is as built.
+static future<> mark_view_schema_as_built(utils::chunked_vector<mutation>& out, std::vector<schema_ptr> schemas, api::timestamp_type ts, service::storage_proxy& sp) {
```

Comment:
should be called mark_view_schemas_as_built, in plural, as it looks for all the view schemas.